xds/src/main/java/io/grpc/xds/internal/certprovider/CertificateProvider.java

```diff
@@ -62,4 +62,12 @@ protected CertificateProvider(Watcher watcher, boolean notifyCertUpdates) {
 
   protected final Watcher watcher;
   protected final boolean notifyCertUpdates;
```

Comment:
protected fields can be private as now you have public accessors.

Reply:
Done